lib/Modules/CinderHooks/CinderHooks.psm1

```diff
@@ -537,6 +482,8 @@ function Invoke-ConfigChangedHook {
                     $status = (Get-Service -Name $svc).Status
                     if($status -eq [System.ServiceProcess.ServiceControllerStatus]::Running) {
                         Restart-Service -Name $svc
+                    } else {
+                        Stop-Service -Name $svc
```

Comment:
I don't understand why we're stopping here. If the service is running, we restart, but if it's not running we stop it?